releases/release-1.31/release-notes/maps/pr-124131-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 124131
+releasenote:
+  text: Endpointslices mirrored from Endpoints by the EndpointSliceMirroring controller
```

Comment:
Is this a bug fix? if so, how about changing it to:
Fixed the bug where if Endpointslices mirrored from Endpoints by the EndpointSliceMirroring controller they would not reconcile if modified.